doc/builddep.rst

```diff
@@ -39,6 +39,11 @@ All general DNF options are accepted, see `Options` in :manpage:`dnf(8)` for det
 ``--srpm``
     Treat arguments as source rpm files.
 
+``--skip-unavailable``
+    Skip build dependencies not available in repositories. All available build dependencies will be installed.
+
+Note that `builddep` command does not honor the `--skip-broken` option so there is no way how to skip uninstallable packages (e.g. with broken dependencies).
```

Comment:
Not important note: What about to replace `Note that` by `Note:`?

Reply:
FWIW I think saying _"Note that"_ is fine :slightly_smiling_face: 

But I'd throw a comma in there and leave out the _"how"_:
> Note that `builddep` command does not honor the `--skip-broken` option**,** so there is no **way to** skip uninstallable packages

Reply:
Thanks Lukas